services/frontend/web_assets/index.html

```diff
@@ -23,13 +23,6 @@
         <h1>Hot R.O.D.</h1>
         <h4><em>Rides On Demand</em></h4>
 
-        <br />
-        <div>
-          Signadot Workspace: <input type="text" id="workspace" />
```

Comment:
I think after this is run, it needs to generate code as well to take effect.
```
esc -pkg frontend -o ./services/frontend/gen_assets.go  -prefix ./services/frontend/web_assets ./services/frontend/web_assets
```

Reply:
Yeah I went ahead and uncommented that line in build.sh since it seems to be fast anyway. I also had to tweak the command, perhaps because of a new `esc` version?